README.md

```diff
@@ -63,6 +64,30 @@ yarn run dev
 
 In this project we are using `yarn` because `netlify` builds our site with `yarn`.
 
+### Folder style guide
+
+W> For new tech topics, you will need to create a new folder.
```

Comment:
I'm curious where the `W>` and `T>` are from. This uses GitHub Flavored Markdown, which doesn't support that. More information on the syntax: https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown

Reply:
https://webpack.js.org/writers-guide/ 
I thought it was universal but I guess not

Reply:
That is some nice formatting! What flavour of markdown would that be?